test/reporter-html-no-qunit-element.js

```diff
@@ -0,0 +1,10 @@
+(function() {
+
+QUnit.module( "reporter-html-no-qunit-element");
+
+QUnit.test( "reporter/html when no qunit element present", function( assert ) {
+	assert.expect( 1 );
+    assert.ok(true);
```

Comment:
use tabs instead of spaces for indentation and missing spaces between parenthesis.